src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,10 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T extends Machine> {
+    int NUMBER_OF_MACHINES = 3;
+
+    List<? extends Machine> get();
```

Comment:
What is the reason for having this **T** if we do not use it?
![image](https://github.com/mate-academy/jv-pecs/assets/20326858/1a4ef031-df74-4135-866b-d2fdbdcf4f02)

Reply:
The logic of using of this restricted generic is that interface will also be used for Machine subclasses. Simulataneously, typical generics are invariant.